<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameBoard</class>
 <widget class="QWidget" name="gameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	border: 2px solid balck;
	border-radius: 5px;
	margin-top: 1ex;
}

QGroupBox::title{
	subcontrol-position: top center;
	padding: 0 5px;
}

QPushButton{
	background-color: black;
 	border-style: solid;
 	border-width:1px;
 	border-radius:10px;
 	border-color: black;
 	max-width:20px;
 	max-height:20px;
 	min-width:20px;
 	min-height:20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mainHBL" stretch="2,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gameGL">
       <property name="spacing">
        <number>1</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="statsVBL" stretch="1,1">
       <item>
        <widget class="QGroupBox" name="playerTurnGB">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Current Player's Turn</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="playerTurnL">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="currentWinningPlayerGB">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Current Winning Player</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="winningPlayerL">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>16</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
